.github/workflows/release-npm.yml

```diff
@@ -61,12 +61,12 @@ jobs:
           fi
           echo $NOCODB_SDK_PKG_NAME
           targetEnv=${{ github.event.inputs.targetEnv || inputs.targetEnv }} targetVersion=${{ github.event.inputs.tag || inputs.tag }} node scripts/bumpNocodbSdkVersion.js &&
-          pnpm --filter=${NOCODB_SDK_PKG_NAME} install --ignore-scripts --no-frozen-lockfile --ignore-workspace && pnpm --filter=${NOCODB_SDK_PKG_NAME} run build && pnpm --filter=nc-plugin install && pnpm --filter=nc-plugin build && pnpm --filter=${NOCODB_SDK_PKG_NAME} publish --no-git-checks &&
+          # pnpm --filter=${NOCODB_SDK_PKG_NAME} install --ignore-scripts --no-frozen-lockfile --ignore-workspace && pnpm --filter=${NOCODB_SDK_PKG_NAME} run build && pnpm --filter=nc-plugin install && pnpm --filter=nc-plugin build && pnpm --filter=${NOCODB_SDK_PKG_NAME} publish --no-git-checks &&
```

Comment:
pnpm --filter=nc-plugin install && pnpm --filter=nc-plugin build
these are required we can move them to another line